java/grpc-client/src/main/java/io/vitess/client/grpc/GrpcClientFactory.java

```diff
@@ -48,6 +50,7 @@
 public class GrpcClientFactory implements RpcClientFactory {
 
   private RetryingInterceptorConfig config;
+  private CallCredentials credentials;
```

Comment:
Minor naming suggestion -- I'd maybe call the variable `callCredentials` to be consistent with the class name and to distinguish from other kinds of credentials.

Reply:
👍 renamed to `callCredentials`